workload-team/cluster-stamp.bicep

```diff
@@ -630,7 +695,11 @@ resource mc 'Microsoft.ContainerService/managedClusters@2024-03-02-preview' = {
     }
     azureMonitorProfile: {
       metrics: {
-        enabled: false // This is for the AKS-PrometheusAddonPreview, which is not enabled in this cluster as Container Insights is already collecting.
+        enabled: true
+        kubeStateMetrics: {
+          metricAnnotationsAllowList: ''
+          metricLabelsAllowlist: ''
```

Comment:
Can you confirm what these are for - what are they allowing/disallowing?

Reply:
added a few comments and links @johndowns, basically this is configuration for kube state metrics

done | address from 11990b9a7ec907bfe856fdb50955fd6c2a211078

If the comments are not desired I could revert this.